misk/src/test/kotlin/misk/web/interceptors/RequestLoggingInterceptorTest.kt

```diff
@@ -354,3 +398,31 @@ internal class ConfigOverrideAction @Inject constructor() : WebAction {
   ) // these values overridden by RequestLoggingConfig binding above
   fun call(@PathParam message: String): String = "echo: $message"
 }
+
+/**
+ * A [RequestLoggingTransformer] that doesn't like seeing "Evan" in String response bodies
+ */
+internal class EvanHatingTransformer @Inject constructor() : RequestLoggingTransformer {
```

Comment:
I disagree with the premise of this transformer 😂 😭

Reply:
I don't condone name-based prejudice but I think it's import for Transformers to be able to express their views.

Reply:
For future maintainers, I'd lean towards renaming this to something less personal.